charts/xchemlab/values.yaml

```diff
@@ -156,6 +156,40 @@ opa-kube-mgmt:
   rbac:
     create: false
 
+postgresql-ha:
+  postgresql:
+    existingSecret: postgres-passwords
+    podAntiAffinityPreset: hard
+    resources:
+      requests:
+        cpu: 500m
+        memory: 512Mi
+      limits:
+        cpu: 2
+        memory: 2Gi
+    initdbScripts:
+      db-init.sql: |
+        SELECT 'CREATE DATABASE compound_library' OWNER xchemlab WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'compound_library')\gexec;
+        SELECT 'CREATE DATABASE compound_soaking' OWNER xchemlab WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'compound_soaking')\gexec;
+        SELECT 'CREATE DATABASE crystal_library' OWNER xchemlab WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'crystal_library')\gexec;
+        SELECT 'CREATE DATABASE targeting' OWNER xchemlab WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'targeting')\gexec;
+        SELECT 'CREATE DATABASE pin_packing' OWNER xchemlab WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'pin_packing')\gexec;
```

Comment:
Are we able to move this to a configmap / secret somehow? Just to tidy up the values file